.github/workflows/build.yml

```diff
@@ -3,6 +3,8 @@ name: Build
 on:
   push:
     branches: [main]
+    tags:
+      - 'v*.*.*'
```

Comment:
i don't recall if that's a regex, but the `.` might need escaping, and potentially a `^` up-front. but since they're tags...

Reply:
This one comes from https://github.com/softprops/action-gh-release#-limit-releases-to-pushes-to-tags, which hopefully has the correct syntax.